command/ssh.go

```diff
@@ -808,12 +809,22 @@ func (c *SSHCommand) parseSSHCommand(args []string) (hostname string, username s
 			continue
 		}
 
+		// If -l has been specified then this is our user name
+		if arg == "-l" {
+			username = i
+			continue
+		}
+
 		// this is an ssh option, lets see if User or Port have been set and use it
 		if arg == "-o" {
-			split := strings.Split(i, "=")
+			split := strings.SplitN(i, "=", 2)
 			key := split[0]
-			// Incase the value contains = signs we want to get all of them
-			value := strings.Join(split[1:], " ")
+			value := split[1]
+
+			// Remove double quotes
+			if value[0:1] == "\"" {
+				value, err = strconv.Unquote(value)
```

Comment:
If you want to ignore the error, please use `_` instead of `err` and provide a comment explaining why it's safe to ignore.  If not, please handle it.